projects/devices/robotiq/protos/Robotiq3fGripper.proto

```diff
@@ -115,16 +116,17 @@ PROTO Robotiq3fGripper [
                           device [
                             RotationalMotor {
                               name "finger_1_joint_3"
-                              maxVelocity 100
-                              maxPosition 3.1416
-                              maxTorque 100
+                              maxVelocity 100.0
+                              minPosition -1.1694
```

Comment:
And here.

Reply:
But this one doesn't match:
https://github.com/ros-industrial/robotiq/blob/66961ec6b6c9c493f13410d16ce32cea1648babf/robotiq_3f_gripper_visualization/cfg/robotiq-3f-gripper_finger_articulated_macro.xacro#L127

Reply:
it's lower - upper
`lower="-1.2217" upper="-0.0523" ` --> `lower="-1.1694" upper="0"   `
together with  line 129, 
`rotation 0 0 1 -0.0523`
this makes it correct

Reply:
Ok, then the idea would be to keep the original limit and instead of putting the rotation to set the `jointparameter.position` field. I will give it a try.